cmd/bb_runner/BUILD.bazel

```diff
@@ -46,6 +46,24 @@ container_layer(
     files = ["@com_github_krallin_tini_tini_static_amd64//file"],
 )
 
+container_layer(
+    name = "install_layer",
+    files = ["install"],
+)
+
+container_image(
+    name = "bb_runner_installer",
+    base = "@busybox//image",
+    cmd = ["/bb"],
+    entrypoint = ["/install"],
+    layers = [
+        ":install_layer",
```

Comment:
What about the passwd layer? Wouldn't we want to bring that one in as well? Or do we expect people to run builds as some other existing user (e.g., `nobody`)?

Reply:
On kubernetes it doesn't seem to be important, and splicing files into /etc (which exists in all the underlying images) is much trickier, we can't just mount over it and it doesn't exist yet when the init container runs. The RBE images have a bunch of users in them already, and other images may have their own user field in the docker manifest - since we're not supplying the container image itself in this model, we can't set it directly. We can only directly influence the user in the init container, which isn't interesting.

Some brief inspection indicates that the ubuntu RBE images come up as uid=0 by default, but we could pick nobody or bin or something in the k8s config.

My considered opinion on this topic is that only the thing which builds the underlying runner image has any hope of solving this problem, so we just stop trying to solve it. In this model bb_runner is a guest injected into this operating system.